### PR TITLE
nodejs client 0.10.0 release

### PR DESCRIPTION
**Pull request by NeerajGartia21**
Blog for the Node.js client 0.10.0.

### CODE REVIEW

**Comment by NeerajGartia21** [comment]
@rigazilla Is it required to add the steps for configuring the client?

File: _posts/2020-09-25-nodejs-010-query.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: blog
+title: Infinispan Node.js client supports query
+permalink: /blog/:year/:month/:day/infinispan-js-client
+date: '2022-09-25T12:00:00.000-00:00'
+author: Neeraj Gartia
```
Comment (rigazilla) [inline _posts/2020-09-25-nodejs-010-query.adoc:6]: @NeerajGartia21 please add a short bio in '_data/authors.yml' and change the author id here accordingly

Comment (NeerajGartia21) [reply _posts/2020-09-25-nodejs-010-query.adoc:6]: sure sir!

**Comment by tristantarrant** [comment]
I added some comments

**Comment by NeerajGartia21** [comment]
Thanks for the changes @tristantarrant!

**Comment by rigazilla** [comment]
merged, thanks @NeerajGartia21 !

**Pull request merged**